galpy/potential/amuse.py

```diff
@@ -98,13 +101,14 @@ def get_potential_at_point(self,eps,x,y,z):
         HISTORY:
            2019-08-12 - Written - Webb (UofT)
         """
-        R= numpy.sqrt(x.value_in(units.kpc)**2.+y.value_in(units.kpc)**2.)
-        zed= z.value_in(units.kpc)
-        phi= numpy.arctan2(y.value_in(units.kpc),x.value_in(units.kpc))
+        R= numpy.sqrt(x.value_in(u.kpc)**2.+y.value_in(u.kpc)**2.)
+        zed= z.value_in(u.kpc)
+        phi= numpy.arctan2(y.value_in(u.kpc),x.value_in(u.kpc))
         return potential.evaluatePotentials(self.pot,R/self.ro,zed/self.ro,
                                             phi=phi,t=self.tgalpy,
-                                            ro=self.ro,vo=self.vo) \
-                                            | units.km**2*units.s**-2
+                                            ro=self.ro,vo=self.vo,
+                                            use_physical=False
+                                            ) | u.km**2*u.s**-2
```

Comment:
Think this needs a ``*self.vo**2`` to get the right units

Reply:
done

Reply:
I've implemented this in a new commit.